configure.ac

```diff
@@ -31,6 +48,7 @@ AC_SUBST(DOMJUDGE_VERSION, $PACKAGE_VERSION)
 AC_DEFINE_UNQUOTED(DOMJUDGE_VERSION, "$PACKAGE_VERSION",
 [DOMjudge version number (alias for PACKAGE_VERSION)])
 
+if test "x$INSTALL_JUDGEHOST" != xno; then
```

Comment:
I didn't check this in detail, but did you test that this works as expected with passing `--disable-judgehost` also?